### PR TITLE
Insertar funcionalidad del Fabric

### PR DESCRIPTION
**Pull request by rbngrc**

### CODE REVIEW

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java [hunk 1]
```diff
@@ -14,7 +18,11 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        Fabric.with(this, new Answers());
         setContentView(R.layout.activity_main);
+        // TODO: Use your own attributes to track content views in your app
+
```
Comment (rbngrc) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java]: Blank line

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java [hunk 2]
```diff
@@ -14,7 +18,11 @@
...
+
```
Comment (rbngrc) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java]: Blank line

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java [hunk 3]
```diff
@@ -14,7 +18,11 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        Fabric.with(this, new Answers());
         setContentView(R.layout.activity_main);
+        // TODO: Use your own attributes to track content views in your app
```
Comment (rbngrc) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java]: Is this usefull??

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java [hunk 4]
```diff
@@ -78,6 +78,7 @@ private void doLogin() {
         String password = passwordTextBox.getText().toString();
 
         apiManager.login(username, password, onLoginResponse);
+
```
Comment (rbngrc) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java:81]: Blank line

Comment (seniorglez) [reply Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java:81]: Visto

**Review by jvillafanez — APPROVED** [review]
👍

**Pull request merged**